drivers/dac/ad3530r/ad3530r.h

```diff
@@ -84,14 +84,14 @@
 /* DAC configuration registers */
 #define AD3530R_REG_ADDR_HW_LDAC_EN_0           (AD3530R_R1B | 0xD0)
 #define AD3530R_REG_ADDR_SW_LDAC_EN_0           (AD3530R_R1B | 0xD1)
-#define AD3530R_REG_ADDR_DAC_CHN(x)             (AD3530R_R2B | (0xD2 + (x * 2)))
+#define AD3530R_REG_ADDR_DAC_CHN(x)             (AD3530R_R2B | (0xD2 + ((x % 8) * 2)))
```

Comment:
i would use `x & 0x7` instead of `x % 8` for performance reasons, especially if these macros are expanded in a loop

Reply:
Agree!